modules/core/src/v2/recovery.ts

```diff
@@ -1,11 +1,22 @@
 import { VirtualSizes } from '@bitgo/unspents';
```

Comment:
Consider adding @prettier pragma at the top if it doesn't mess up the formatting too much.

Reply:
just add:
/**
* @prettier
*/
??